README.md

```diff
@@ -6,6 +6,13 @@ passwords, it is required now to do a more complex kind of authorization which r
 on tokens. And that's usually done through the OAuth2
 [Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/) flow.
 
+<table><td>:memo: **Note**:
+This more complex authorization flow must be done once via an attended run, but once
+completed, your automation using the final vault objects will be able to send and
+receive emails unattendedly. We recommend you use this portal bot to complete the
+initial setup within your workspace.</td></table>
```

Comment:
```suggestion
<table><td>:memo: Note:
This more complex authorization flow must be done once via an attended run, but once
completed, your automation using the final token from the Vault will be able to send and
receive e-mails unattendedly. We recommend you use this Portal robot to complete the
initial setup within your workspace.</td></table>
```

- The **Note** strong emphasis
- Small Vault reformulation

Since it was rendered like (no full Markdown support I believe):
<img width="870" alt="Screenshot 2023-06-15 at 10 56 26" src="https://github.com/robocorp/example-oauth-email/assets/709053/4523bfaf-5930-4d23-9eea-082414142316">